Gemfile

```diff
@@ -24,6 +24,7 @@ end
 
 group :test do
   gem "ammeter"
+  gem "rails-controller-testing"
```

Comment:
We've previously had some problems with this gem and compatibility across different Rails versions. Do you think there's a way that we could pull off testing for this without bringing it back in?